src/routes/index.js

```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
 
 const searchHome = require('./search_home.js');
+const startupProfile = require('./startup_profile.js')
 
 router.get('/', searchHome);
+router.get('/Startup_Profile', startupProfile);
```

Comment:
So we should keep the naming of endpoints consistent for example in my PR the form endpoint is 'registerStartup'